### DISCUSSION TITLE
Confetti timed limit

### DISCUSSION THREAD

**Discussion by garettharper** [matteobruni/tsparticles#1538] [Q&A] [answered]
Is there a way to limit the animation to 5 seconds? I

**Comment by matteobruni** [comment]
Yes, you can add this

``` javascript 
life: {
    duration: 5,
    count: 1
}
```

to each emitter options

**Comment by garettharper** [comment, marked as answer]
Wow, that was fast! 

I've adjusted the life and it does stop after 5 secs, but the 'bursts' are much faster and no longer randomly displayed across the screen.  Can you take a look to see what I'm missing? Making the change on codepen does the same -->  https://codepen.io/matteobruni/pen/JjEddbQ

https://www.eku.edu/_uat/confetti.html
https://templates.eku.edu/_resources/js/confetti.js

**Reply by matteobruni** [reply]
I thought you were talking about the confetti cannon and not the explosions.

You can set life `count` to `7` instead of `0` (`5` seconds, `0.6` delay, `0.1` duration -> `5 / (0.6 + 0.1) = 7.14` so `7` lives) and your effect will last around `5` (`4.9`) seconds.

So something like this

``` javascript
life: {
    duration: 0.1,
    count: 7,
    delay: 0.6
}
```

**Comment by garettharper** [comment]
This is perfect! Thanks, Matteo!!
